behave/model.py

```diff
@@ -1803,6 +1808,7 @@ def run(self, runner, quiet=False, capture=True):
                 #  * Even EMPTY multiline text is available in context.
                 runner.context.text = self.text
                 runner.context.table = self.table
+                runner.context.attach = attach
```

Comment:
Is that the right way?

All other predefined `context` attributes are (kind of) read-only members, but this one is a method. It feels like this is trying to escape from a clean image, a clean approach. Or do I get something wrong?

Reply:
Thanks for the review! I can move this method to `runner.py` and add it to the `Context` class. That would be better.

Reply:
I updated the code, what do you think?